gateway/src/api/latest.rs

```diff
@@ -1364,6 +1385,23 @@ pub mod tests {
         Ok(())
     }
 
+    #[test_context(TestProject)]
+    #[tokio::test]
+    async fn api_delete_project_that_is_ready(project: &mut TestProject) -> anyhow::Result<()> {
+        project.router_call(Method::DELETE, "/delete").await;
+
+        Ok(())
+    }
```

Comment:
Clean! :heart_eyes: Can we apply these utilities for the existing tests as well, and break them up into smaller pieces (looking at you `api_create_get_delete_projects`)? We can do it in a later PR to not block this for tomorrows release.

Reply:
Yip, getting to them in another PR is the plan.... and maybe even adding them to the end of CI :smile: